integration/hurl/tests_ok/follow_redirect.hurl

```diff
@@ -43,6 +49,8 @@ Authorization: Basic Ym9iQGVtYWlsLmNvbTpzZWNyZXQ=
 HTTP 200
 [Asserts]
 redirects count == 1
+redirects nth 0 location == "http://localhost:8000/followed-redirect-basic-auth"
```

Comment:
The URLs with `change_host=true` change their host during the redirections from localhost to 127.0.0.1 so we have to adapt these tests (that's why the CI is failing):

```
redirects nth 0 location == "http://127.0.0.1:8000/followed-redirect-basic-auth"
```